ndmg/stats/qa_skullstrip.py

```diff
@@ -0,0 +1,244 @@
+    idx = 0
+    if edge:
+        min_val = 0
+        max_val = 1
+    else:
+        min_val, max_val = get_min_max(b0, minthr, maxthr)
+
+    for i, coord in enumerate(coords):
+        for pos in coord:
+            idx += 1
+            ax = foverlay.add_subplot(3, 3, idx)
+            ax.set_title(var[i] + " = " + str(pos))
+            if i == 0:
+                image = ndimage.rotate(b0[pos, :, :], 90)
+                atl = ndimage.rotate(original[pos, :, :], 90)
+            elif i == 1:
+                image = ndimage.rotate(b0[:, pos, :], 90)
+                atl = ndimage.rotate(original[:, pos, :], 90)
+            else:
+                image = ndimage.rotate(b0[:, :, pos], 0)
+                atl = ndimage.rotate(original[:, :, pos], 0)
+
+            if idx % 3 == 1:
+                ax.set_ylabel(labs[i])
+                ax.yaxis.set_ticks([0, image.shape[0] / 2, image.shape[0] - 1])
+                ax.xaxis.set_ticks([0, image.shape[1] / 2, image.shape[1] - 1])
+            if edge:
+                image = edge_map(image).data
+                image[image > 0] = max_val
+                image[image == 0] = min_val
+            # Set the axis invisible
+            plt.xticks([])
+            plt.yticks([])
+
+            # Set the frame invisible
+            ax.spines['top'].set_visible(False)
+            ax.spines['right'].set_visible(False)
+            ax.spines['bottom'].set_visible(False)
+            ax.spines['left'].set_visible(False)
+            ax.imshow(atl, interpolation="none", cmap=cmaps[0], alpha=0.9)
+            ax.imshow(
+                opaque_colorscale(
+                    cmaps[1], image, alpha=0.9, vmin=min_val, vmax=max_val
+                )
+            )
+            if idx ==3:
+                plt.plot(0, 0, "-", c="magenta", label='skull')
+                plt.plot(0, 0, "-", c="green", label='brain')
+                # box = ax.get_position()
+                # ax.set_position([box.x0, box.y0, box.width, box.height*0.8])
+                plt.legend(loc='best', fontsize=15, frameon=False, bbox_to_anchor=(1.5, 1.5))
+
+    # Set title for the whole picture
+    [a, b, c] = ori_shape
+    title = 'QA For skullstrip. Scan Volume:' + str(a) + '*' + str(b) + '*' + str(c)
+    foverlay.suptitle(title, fontsize=24)
+    foverlay.set_size_inches(12.5, 10.5, forward=True)
+    return foverlay
+
+def get_true_volume(nparray):
+    """
+    locates 1/4,1/3,1/2 slices of brain and returns them
+    """
+    img_arr = nparray.astype(int)
+    threshold = int(1)
+    img_arr[img_arr <= threshold] = 0
+    img_arr[img_arr > threshold] = 1
+    true_volume = np.where(img_arr == 1)
+    x = get_range(true_volume, 0)
+    y = get_range(true_volume, 1)
+    z = get_range(true_volume, 2)
+    return x, y, z
+
+def get_range(array,i):
+    min_num = min(array[i])
+    max_num = max(array[i])
+    arrange = np.arange(min_num, max_num)
+    quarter = np.percentile(arrange, [25, 50, 75]).astype(int)
```

Comment:
no docstrings at all here. Please add them, and in the same format the rest of ndmg has.

Reply:
fix it, but need help to check

Reply:
Where is the commit that you fixed it? It doesn't look fixed on your most recent commit: https://github.com/NeuroDataDesign/ndmg/blob/qa_skullstrip_pr/ndmg/stats/qa_skullstrip.py#L189

Reply:
in here
https://github.com/neurodata/ndmg/pull/366/commits/e91ce6433965074c38e7cc7e769c41793903f074